opal/core/application.py

```diff
@@ -142,7 +142,7 @@ def get_menu_items(klass, user=None):
                 items.append(logout)
                 if user.is_staff:
                     items.append(admin)
-        return items
+        return [item for item in items if item.for_user(user)]
```

Comment:
I think we want an explicit test for `Application.get_menu_items` somewhere near here: https://github.com/openhealthcare/opal/blob/v0.12.0/opal/tests/test_core_application.py#L65